src/views/Home/RecentActivity/styles.scss

```diff
@@ -0,0 +1,52 @@
+.recent-activity {
+    display: flex;
+    flex-direction: column;
+
+    .header {
+        padding: var(--dui-spacing-small) 0;
+    }
+
+    .content-container {
+        display: flex;
+        flex-direction: column;
+        border-radius: 10px;
+        background-color: var(--dui-color-negative);
+        height: 420px;
+        overflow-y: auto;
+
+        .activity-list {
+            flex-grow: 1;
+            overflow-y: auto;
+
+            .activity-item {
+                display: flex;
+                align-items: center;
+                border-bottom: var(--width-separator-thin) solid var(--dui-border-color-card);
+                padding: calc(var(--dui-spacing-extra-large) - var(--dui-spacing-small));
+
+                .display-picture {
+                    margin-right: var(--spacing-medium);
+                    border-radius: 50%;
```

Comment:
Isn't this done internally by DisplayPicture

Reply:
This is what it looks if we don't use a wrapper
![image](https://user-images.githubusercontent.com/51225380/112419451-8b12d000-8d53-11eb-9800-75c4a142bb89.png)